Completed cheet sheet.

codyeatworld opened the pull request:
🤘 🤘 🤘 🤘 🤘 🤘 🤘 🤘 🤘 🤘 🤘 🤘 🤘 🤘

codyeatworld committed:
Completed cheet sheet.

dancinturtle commented:
This is great, very thorough! One thing I noticed in your lists.py:
people = ["Rick", "Morty", "Summer", "Beth", "Jerry"]
min(people)
You have commented that this would return "Rick", but it returns "Beth". Make sure you understand why.
Another thing in strings.py:
fruits = "Bananas, Mangos, Pineapples."
fruits.split(' ')
You commented that this would return  ['Bananas', 'Mangos', 'Pineapples.'], however, you're splitting on the space. What happens to your commas?

codyeatworld commented:
Definitely looking into the min functionality more in depth now. 

I think I removed the comma for the split function while explaining it to classmates, it was there in the beginning. 😄 

The comma/space is the delimiter and split removes it, to add it back, you would use the join method with the same delimiter.

Thanks for the quick review!
